tests/lib/LMSConfig/ConfigContainerTest.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace LMS\Tests;
```

Comment:
dlaczego rezygnujemy z własnej przestrzeni nazw `LMS\Tests`. Jak dla mnie to akurat wygląda jak krok w odwrotnym kierunku niż podąża cały świat.

Reply:
Ok - zostawiłem przestrzeń nazw.

Reply:
Zobaczymy czy będzie travis na zielono.